### ISSUE TITLE
Defining Field for array of string values prop

### ISSUE THREAD

**Issue by sagarchoudhary96** [measuredco/puck#61]
My component have a prop tags of array type but values as string for it ``tags: string[]` . how can i define Field config for this prop, seems like it's currently not supported.

**Comment by Danm72** [comment]
If i'm understanding you correctly you can add this to your fields config:
`tags: {
            type: "array",
            getItemSummary: (item) => item.tag || "Tag",
            arrayFields: {
                tag: {type: "text"},
            },
        },`

**Comment by chrisvxd** [comment]
Correct, we only support objects in `array` fields at the time-being, so the easiest thing to do is as @Danm72 proposed.

If your component expects `string[]`, suggest you could wrap your component in another one to map from the object to string.
